plugin/src/main/java/org/wildfly/plugin/provision/PackageServerMojo.java

```diff
@@ -271,6 +279,22 @@ public class PackageServerMojo extends AbstractProvisionServerMojo {
     @Parameter(alias = "bootable-jar-install-artifact-classifier", property = PropertyNames.BOOTABLE_JAR_INSTALL_CLASSIFIER, defaultValue = BootableJarSupport.BOOTABLE_SUFFIX)
     private String bootableJarInstallArtifactClassifier;
 
+    /**
+     * A list of the dependencies to include as deployments. The dependencies must be defined as dependencies in the
```

Comment:
We should perhaps add the syntax of the dependency?

Reply:
Good call. I meant to add that and completely forgot.